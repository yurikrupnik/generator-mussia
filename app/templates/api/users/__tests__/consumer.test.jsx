import React from 'react';
import Consumer from '../consumer';

const {
    describe,
    test,
    expect,
    render
} = global;

describe('theme consumer', () => {
    test('render consumer with render props', () => {
        const wrapper = render( // notice render here
            <Consumer render={(props) => {
                const { fetch } = props;
                fetch();
                return (
                    <div>
                        asd
                    </div>
                );
            }}
            />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
