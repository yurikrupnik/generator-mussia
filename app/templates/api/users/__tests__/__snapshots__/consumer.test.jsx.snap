// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme consumer render consumer with render props 1`] = `
<div>
  asd
</div>
`;
